tests/sentry/api/endpoints/release_thresholds/test_release_threshold_status.py

```diff
@@ -814,3 +814,141 @@ def test_multiple_environments_within_timeseries(self):
             "window_in_seconds": 60,  # NOTE: window_in_seconds only used to determine start/end. Not utilized in validation method
         }
         assert not is_error_count_healthy(ethreshold=threshold_unhealthy, timeseries=timeseries)
+
+    def test_unordered_timeseries(self):
+        """
+        construct a timeseries with:
+        - a single release
+        - a single project
+        - no environment
+        - multiple timestamps both before and after our threshold window
+        - all disorganized
+        """
+        now = datetime.utcnow()
+        timeseries = [
```

Comment:
Is everything in this timeseries like a poll value? So we're polling every minute for the last 3 minutes and now, and the count of each poll is the number of errors

Reply:
Not exactly

everything in the timeseries is just a summation of events over a window

When someone polls our api - we're grabbing a sum of those values that falls within our desired polling window - and comparing that summed value against our threshold